lib/tax_cloud/tax_codes.rb

```diff
@@ -1,7 +1,7 @@
 module TaxCloud #:nodoc:
   class TaxCodes
     # General purpose tax code.
-    GENERAL = 00000
+    GENERAL = 0o0000
```

Comment:
Is this accidental? Or does this mean something?

Reply:
This was accidental. I'll review the rest of that commit, which I thought was just auto-generated by rubocop tomorrow morning.

Reply:
Actually that probably should be a quoted string to start with if it's a TIC. The change *probably was*  generated by rubocop because it's weird to have a bare integer that's all zeros.

Reply:
I think Rubocop thinks this is binary number cause why would you do 0000 otherwise. Thanks for looking at it.